neptune/management/exceptions.py

```diff
@@ -26,3 +26,13 @@ def details(self):
             "code": self.code,
             "description": self.details,
         }
+
+
+class MissingWorkspaceName(ManagementOperationFailure):
+    code = -2
+    description = "Missing workspace name"
```

Comment:
let's try a little more verbose, please ;) maybe something like:
`Cannot resolve project {name}, you have to provide a workspace name.`